modular_skyrat/modules/modular_implants/code/nifsofts/prop_summoner.dm

```diff
@@ -87,10 +89,49 @@
 //Summonable Items
 ///A somehow wekaer version of the toy katana
 /obj/item/toy/katana/nanite
+	name = "hexblade"
+	desc = "One of the first groups to contribute to the Caeruleam Grimoire's repository were the Malatestan Duelists, a group of mercenary-philosophers seeking to become undisputed masters of the principal art of Cutting. Originally intended as a means of generating perfectly sharp, perfectly unbreakable, and perfectly capable of the Sanctioned Action: to cut. However, these 'blunted' prop Icons are only a mere shadow of what the Duelists originally developed, the only version of the Icon permitted by interstellar law to civilians; normally seen on convention floors or in the hands of those wishing to spar without risk."
 	force = 0
 	throwforce = 0
 
-///A nanite version of the cane.
+/obj/item/storage/dice/nanite
+	name = "dice set"
+	desc = "A gorgeous replication of a gorgeous set of dice. These were modeled after a set of dice originally in the possession of the Selenian Wargaming Society, carved from rare lunar crystals over two hundred years ago. While no one but the Prime Game Master may ever roll even a single piece from the original set, the Society has graciously donated virtual replicas to the Altspace Coven's repo, as a token of their appreciation for aid in more live-action forms of roleplay."
+
+/obj/item/toy/cards/deck/nanite
+	name = "main deck"
+	desc = "Another piece of gaming equipment graciously donated from the Selenian Wargaming Society, these cards employ a localized field of near-invisible nanites equipped with advanced eye tracking software; to ensure the display on the cards does not allow for peeking. Additionally, over five hundred thousand variations of the standard fifty two card deck are supported, in all known forms of writing. Lastly, a collaboration with the Altspace Coven has yielded a deck of tarot cards; the witches designing it claiming that the incorporeal nature of the cards allow them a higher connectivity with fate itself. "
+
+/obj/item/toy/cards/deck/tarot/nanite
+	name = "tarot deck"
+	desc = "Another piece of gaming equipment graciously donated from the Selenian Wargaming Society, these cards employ a localized field of near-invisible nanites equipped with advanced eye tracking software; to ensure the display on the cards does not allow for peeking. Additionally, over five hundred thousand variations of the standard fifty two card deck are supported, in all known forms of writing. Lastly, a collaboration with the Altspace Coven has yielded a deck of tarot cards; the witches designing it claiming that the incorporeal nature of the cards allow them a higher connectivity with fate itself. "
```

Comment:
You copy-pasted this description but forgot that only playing card decks have 52 cards. Tarot decks have 72.

Reply:
this is a placeholder